client/ayon_core/tools/launcher/ui/hierarchy_page.py

```diff
@@ -49,12 +49,18 @@ def __init__(self, controller, parent):
         folders_filter_text = PlaceholderLineEdit(folders_wrapper)
         folders_filter_text.setPlaceholderText("Filter folders...")
 
+        show_only_my_assignments = QtWidgets.QCheckBox(
+            "Show only my assignments"
+        )
+        show_only_my_assignments.setChecked(True)
+
```

Comment:
Admittedly now it starts feeling weird there's so much duplicated code for this hierarchy widget that also allows filtering across the codebase - but hey, we likely have good reasons for it :P

Reply:
yeah... I felt a bit like playing whac-a-mole trying to add it on all the places